functions.py

```diff
@@ -94,6 +94,70 @@ def get_bugs_set(blockers):
 	return bug_set
 
 
+def get_jenkins_api_info(server, job_name):
```

Comment:
nit: The name is misleading and IMO should be something like `get_jenkins_job_info` to represent what is actually returned